### ISSUE TITLE
Create UI Homescreen

### ISSUE THREAD

**Issue by omorkved** [uchicago-cs/chiventure#205]
Create a user-friendly homescreen

- Chiventure logo should appear when first loaded
- instructions should appear "press _something_ for help, press _something_ to show the map..."

etc.

**Comment by sebvil** [comment]
In order to be able to use the Unicode characters used in the banner, we must call the function `    setlocale(LC_ALL, "")` (must include locale.h) before `initscr` is called. Thus, this is done in the ui context struct initialization. Furthermore, the we must use `lncursesw` instead of `-lncurses` in order to support the utf-8 locale.

**Comment by sebvil** [comment]
Closing Statement:  Printing the home screen was not as trivial as it seemed at first. The ncurses standard library does not support for Unicode characters present in the banner, so when linking, we had to change `-lncurses` to `lncursesw`. Additionally, I had to call the command `setlocale(LC_ALL, "")` before `initscr` to support the utf-8 locale.. 

After this task, when `chiventure` is run, one sees the banner centered  on the top window of the UI, with the message "Type 'HELP' to show help menu" under it.
